Formula/d/deno.rb

```diff
@@ -43,18 +43,18 @@ class Deno < Formula
   # We use the crate as GitHub tarball lacks submodules and this allows us to avoid git overhead.
   # TODO: Remove this and `v8` resource when https://github.com/denoland/rusty_v8/issues/1065 is resolved
   resource "rusty-v8" do
-    url "https://static.crates.io/crates/v8/v8-0.74.1.crate"
-    sha256 "1202e0bd078112bf8d521491560645e1fd6955c4afd975c75b05596a7e7e4eea"
+    url "https://static.crates.io/crates/v8/v8-0.75.0.crate"
+    sha256 "f9be435abe79a8427b0969f1ac0a3c0e91644235f68a3de5da4a27ec69666985"
   end
 
   # Use the latest tag in https://github.com/denoland/v8/tags.
   resource "v8" do
-    url "https://github.com/denoland/v8/archive/refs/tags/11.7.439.2-denoland-9fcf117b2e045094627c.tar.gz"
-    sha256 "20661cff2d5f5fe7393d992225f7098bede350bc9415068a811ef08e95205b1b"
+    url "https://github.com/denoland/v8/archive/refs/tags/11.7.439.5-denoland-137fc2a297b0f8cf9b1c.tar.gz"
```

Comment:
As a curiousity -- why not use `V8_FROM_SOURCE` for `rusty_v8`?

Reply:
I think it might be the same rationale in the other PR, we want to control the resource, so being explicit rather than being implicit, does that make sense?

Reply:
I think so. I have an idea how it might be solved upstream. I'll play around with some ideas.

Reply:
Thanks for your assistance, @mmastrac.

Reply:
Glad we could get this working!